src/services/analysisService.ts

```
@@ -43,20 +50,104 @@ export async function analyzeError(
     return `Could not analyze the error with Neurolink.${details}`;
   }
 }
 
+/**
+ * Strip ANSI escape codes from a string
+ * These codes are used for terminal colors/formatting and break pattern matching
+ */
+function stripAnsi(str: string): string {
+  // Match ANSI escape sequences: ESC[ followed by parameters and a letter
+  // eslint-disable-next-line no-control-regex
+  return str.replace(/\x1B\[[0-9;]*[a-zA-Z]/g, '');
```

Biome (2.3.13)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)